src/utils/fsUtil.ts

```diff
@@ -42,12 +42,10 @@ const addFileCommands = {
     addFileCommand: 'cat > [FILEPATH] << EOF [FILECONTENT]EOF'
   },
   powershell: {
-    addFileCommand: `@"[FILECONTENT]"@ | Out-File -FilePath "[FILEPATH]"
-    `
+    addFileCommand: `@"[FILECONTENT]"@ | Out-File -FilePath [FILEPATH]`
```

Comment:
I believe we included quotes in case we'd have a file path with a space. Is there a particular reason to remove them?

Reply:
I observed that the filepath already includes the quotes. When I ran the command without the changes within the PR then I received the following suggestion

``` powershell
@"
require('@rushstack/eslint-config/patch/modern-module-resolution');
module.exports = {
  extends: ['@microsoft/eslint-config-spfx/lib/profiles/react'],
  parserOptions: { tsconfigRootDir: __dirname }
};
"@ | Out-File -FilePath "".eslintrc.js""
```

This fails in PowerShell because of the double double quotes around the filepath😄

Reply:
Thanks for confirming. Next time, let's mention that in the commit notes to make it clear upfront 👍

Reply:
Will do!